protocol/messenger_activity_center.go

```diff
@@ -694,3 +697,72 @@ func (m *Messenger) HandleSyncActivityCenterCommunityRequestDecision(state *Rece
 	resp.SetActivityCenterState(s)
 	return nil
 }
+
+func (m *Messenger) HandleFeedItem(feedItem *gofeed.Item) (*MessengerResponse, error) {
```

Comment:
Would it make sense to move these new functions to `messenger_news_feed.go`?

Reply:
You mean I should create a new file? If so, yeah makes sense